jobserver/templates/project_settings.html

```diff
@@ -85,27 +86,64 @@ <h3>Invitations</h3>
     <form method="POST" class="mb-4">
       {% csrf_token %}
 
-        {% if form.non_field_errors %}
-        <ul>
-          {% for error in form.non_field_errors %}
-          <li class="text-danger">{{ error }}</li>
-          {% endfor %}
-        </ul>
-        {% endif %}
+      {% if form.non_field_errors %}
+      <ul>
+        {% for error in form.non_field_errors %}
+        <li class="text-danger">{{ error }}</li>
+        {% endfor %}
+      </ul>
+      {% endif %}
 
-      <div class="d-flex justify-content-between align-items-center">
-        <select id="id_users" name="users" class="w-100 mr-3" multiple required>
+      <div class="form-group">
+        <select id="id_users" name="users" class="w-100" multiple required>
           {% for value, label in form.fields.users.choices %}
           <option value="{{ value }}">{{ label }}</option>
           {% endfor %}
         </select>
 
+        {% for error in form.users.errors %}
+        <p class="text-danger">{{ error }}</p>
+        {% endfor %}
+      </div>
+
+      <div class="form-group mb-3">
+        <p>Select one or more Roles for the User(s) you are inviting.</p>
```

Comment:
My personal preference is not to capitalise Role and User here, but I'll shut up about this if you like.

Reply:
You're literally the only person keeping me on the straight and narrow with this, please don't stop! 😂